manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SecureMixLLC.GlassWireSetup
+PackageVersion: 3.3.499.0

```

```suggestion
PackageVersion: 3.3.499
```
Should Match Control Panel